packages/orm/src/createOrUpdatePartial.ts

```diff
@@ -29,7 +29,15 @@ type AllowRelationsToBeIdsOrEntitiesOrPartials<T> = {
   [P in keyof T]: T[P] extends NullOrDefinedOr<infer U>
     ? U extends Array<infer V>
       ? V extends Entity
-        ? Array<V | (DeepPartialOrNull<V> & { delete?: boolean | null; remove?: boolean | null }) | IdOf<V>> | null
+        ? Array<
+            | V
+            | (DeepPartialOrNull<V> & {
+                delete?: boolean | null;
+                remove?: boolean | null;
+                op?: "remove" | "delete" | "include";
```

Comment:
I went with `include` instead of `add` b/c they're sets so like sending in `op: add` multiple times isn't really re-adding it... :thinking: